src/store/actions/socialActions/FacebookActions.js

```diff
@@ -0,0 +1,40 @@
+import axios from 'axios';
+import {
+  FACEBOOK_AUTHENTICATION,
+  FACEBOOK_AUTH_SUCCESS,
+  FACEBOOK_AUTH_FAIL,
+} from 'store/actions/socialTypes';
+import loginSuccess from 'store/actions/loginActions';
+import { isAuthenticated } from 'utils';
+
+const passToken = token => ({
+  user: {
+    auth_token: token
+  }
+});
+
+const facebookSuccess = resp => ({
+  type: FACEBOOK_AUTH_SUCCESS,
+  payload: resp.data.user.auth_token,
+});
+
+const facebookFail = error => ({
+  type: FACEBOOK_AUTH_FAIL,
+  payload: error.response.data.errors,
```

Comment:
After destructuring in the .catch below this line will need to be changed to
`
const facebookFail = error => ({
  type: FACEBOOK_AUTH_FAIL,
  payload: errors,
`